docs/tutorials/elasticdl_on_prem_cluster.md

```diff
@@ -69,3 +69,48 @@ elasticdl train \
  --image_pull_policy=Always \
  --namespace=kubemaker
 ```
+
+## Add Cluster-specific Information
+
+If the on-prem cluster requires additional specifications to pods or services,
```

Comment:
on-prem => on-premise

Reply:
Usually, the on-premises clusters wouldn't require additional YAML labels.  It is the tailored version of Kubernetes who requires additional labels.

Reply:
Revised.